src/design-tokens/tier-1-definitions/fonts.css

```diff
@@ -46,3 +46,18 @@
   font-style: normal;
   src: url('./fonts/MintGroteskV1.1-Medium.woff2') format('woff2');
 }
+
+/* TODO: move these into .storybook/ and update build:storybook to load these static files as well */
```

Comment:
Does this sound like the right way to get this to work?

Reply:
Probably needs to import this in `.storybook/main.js` as a static file but seems like `staticDirs` config is [broken in Chromatic](https://github.com/chromaui/chromatic-cli/issues/462)

Reply:
Made a ticket: https://czi-tech.atlassian.net/browse/EDS-740